niworkflows/utils/images.py

```diff
@@ -297,3 +297,20 @@ def nii_ones_like(in_file, value, dtype, newpath=None):
     nii.to_filename(out_file)
 
     return out_file
+
+
+def _bspline_grid(in_file):
```

Comment:
Inspiration for using slice number to determine grid sparsity: 
https://itk.org/pipermail/community/2014-February/005036.html

Reply:
Please add the link to the function's documentation